docs/servers/create-gcp-server.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Create a GCP Server
```

Comment:
```suggestion
title: Add a Server to GCP (Google Cloud Platform)
```
Cause the server does not exist yet, and it's going to be provisioned by Devopness itself.

Reply:
what about:
https://github.com/devopness/devopness/blob/2757da9e303846046d1b1ed042742ae8f401bd75/docs/servers/add-gcp-server.md?plain=1#L2
?